packages/apache_tomcat/data_stream/access/manifest.yml

```
@@ -36,6 +36,22 @@ streams:
         required: false
         show_user: false
         description: Processors are used to reduce the number of fields in the exported event or to enhance the event with metadata. This executes in the agent before the logs are parsed. See [Processors](https://www.elastic.co/guide/en/fleet/current/elastic-agent-processor-configuration.html) for details.
+      - name: harvester_limit
+        type: integer
+        title: Harvester Limit
+        description: "Limits the number of harvesters that are started in parallel."
```

This one does not have the link to the documentation like the others.

I wonder if 'harvester' is clear enough for our users. Maybe we could re-word it a bit:
```suggestion
        description: "Limits the number of files that are ingested in parallel."
```

---

@belimawr 
How about: Limits the number of files that are ingested/harvested in parallel?

---

No need the quotation marks("") in the description.